src/apps/vpn/notificationhandler.cpp

```diff
@@ -116,7 +116,7 @@ void NotificationHandler::showNotification() {
         }
 
         QString localizedPreviousExitCountryName =
-            vpn->serverCountryModel()->localizedCountryName(
+            vpn->serverCountryModel()->countryName(
```

Comment:
Here you want: `vpn->currentServer()->localizedPreviousExitCountryName()`

!!! We need to localize the previous country code too :) Can you add a new property and a new method?